### DISCUSSION TITLE
To find number of days

### DISCUSSION THREAD

**Discussion by Niyanthan** [fbaligand/kibana-enhanced-table#227] [Q&A] [answered]
I have a list of user dataset which contains User id and its Created date. I have created the Minimum of Created Date (col1) column for every user. 
Now I need to find the number of days  from the Minimum of Created Date to till date .
I added a computed column with "now" (col2) and in another computed column I tried to find the difference between "col2-col1"
I couldn't get any result

Note that the col2 was  in epoch time format

How to arrive it?

**Comment by fbaligand** [comment, marked as answer]
Well, first, you can get current time with `now()` function in a computed column formula.
All available functions are documented here: https://github.com/fbaligand/kibana-enhanced-table#available-functions
Is your column 1, a "date" column, I mean a column where the value is really a date object, not a string containing a date? It is required to compute date difference.
Then, you don't need to create an intermediate column with "now".
Given that col1 is really a date object (and so column value is at Epoch format in milliseconds), you can use this formula to compute difference from now in milliseconds: `now() - col1`.
And if you want the difference in days, you can use this formula: `(now() - col1) / 1000 / 60 / 60 / 24`

**Reply by Niyanthan** [reply]
Hey Thank you I got my solution

**Reply by fbaligand** [reply]
You’re welcome ;)
